Header header
geometry_msgs/Pose[] poses
geometry_msgs/Point[] nrDetectedLShapes
geometry_msgs/Point[] idx
geometry_msgs/Point[] LShapesIdxs
geometry_msgs/Point[] LShapesPos
geometry_msgs/Quaternion[] LShapesOri
